Snakemake/rules/prepare_files_for_tsinfer.smk

```diff
@@ -93,9 +129,10 @@ rule change_infoAA_vcf:
         HEADERNUM=$(( $(grep "##" {input.vcf} | wc -l) + 1 ))
         INFOLINE=$(( $(grep -Fn "INFO" {input.vcf} | cut --delimiter=":" --fields=1  | head -n1) ))
         awk -v HEADER=$HEADERNUM -v INFO=$INFOLINE 'NR==FNR{{{{a[FNR] = $2; next}}}} FNR<=HEADER{{{{print}}}}; \
-        FNR==INFO{{{{printf "##INFO=<ID=AA,Number=1,Type=String,Description="Ancestral Allele">\\n"}}}}; \
-        FNR>HEADER{{{{$8=a[FNR-HEADER]; print}}}}' OFS="\t" {input.ancestralAllele} {input.vcf} > {output}
+        FNR==INFO{{{{printf "##INFO=<ID=AA,Number=1,Type=String,Description=Ancestral Allele>\\n"}}}}; \
+        FNR>HEADER{{{{$8="AA="a[FNR-HEADER]; print}}}}' OFS="\t" {input.ancestralAllele} {input.vcf} > {output}
         """
+
 rule compress_vcf:
     input:
         rules.change_infoAA_vcf.output
```

Comment:
We are generating A LOT of intermediate files. Is it better to create a subfolder and dump them all there? Or just delete them? I don't know how this would affect the snakemake performance...

Reply:
Done! I've created a directory called /tmp within the Tsinfer directory.